src/finder.js

```diff
@@ -109,6 +111,15 @@ export function find (node, args, raw, minimist) {
   }
 }
 
-export function run ({command, args}) {
-  return command.handler.apply(this, args)
+export function run ({command, args}, root) {
+  if (typeof command.handler === 'function') {
+    return command.handler.apply(this, args)
+  } else if (typeof command.handler === 'string') {
+    return require(path.join(root, command.handler)).default.apply(this, args)
```

Comment:
Should use an util that checks for `req.default || req` for interop with `module.exports = function(){}`.